CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# Changelog
+
+## 1.1.0
+ 
+- Added Mattermost integration: format and send daily paper digests to Mattermost channels.
```

Comment:
Please make subtitles "Added", "Changed" and put bullet points in the corresponding lists